src/averageAges.js

```diff
@@ -20,6 +20,23 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const manCount = people.filter(function(person) {
```

Comment:
```suggestion
  const manCount = people.filter(person =>
```
create arrow functions

Reply:
and fix it everywhere, not only just here